pyproject.toml

```diff
@@ -91,6 +91,7 @@ dependencies = [
     "posthog~=3.21.0",
     "vobject~=0.9.7",
     "pycountry~=24.6.1",
+    "akismet~=24.11.0"
```

Comment:
Is this required? 

Seems like it's a simple REST API call based on docs: https://akismet.com/developers/detailed-docs/comment-check/ 


Also IMO since this is dependent on third-party API (w/ paid plan) it's better if it lives in a marketplace app instead of the framework itself.

Should be easy to hook into comment's insert from a separate app.

Reply:
Once we have a separate blogging module, then it can reside in that app. 

FYI  @sokumon